content/docs/deployment/csmoperator/troubleshooting.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Troubleshooting"
+linkTitle: "Troubleshooting"
+weight: 3
+Description: >
+  Troubleshooting guide
+---
+
+## Frequently Asked Questions
+
+  - [Can CSM Operator manage existing drivers installed using Helm charts or the Dell CSI Operator?](#can-csm-operator-manage-existing-drivers-installed-using-helm-charts-or-the-dell-csi-operator)
+  - [Why does some of the Custom Resource fields show up as invalid or unsupported in the OperatorHub GUI?](#why-does-some-of-the-custom-resource-fields-show-up-as-invalid-or-unsupported-in-the-operatorHub-gui)
+  - [How can I view detailed logs for the CSM Operator?](#how-can-i-view-detailed-logs-for-the-csm-operator)
+
+### Can CSM Operator manage existing drivers installed using Helm charts or the Dell CSI Operator?
+The Dell CSM Operator can't manage any existing driver installed using Helm charts or the Dell CSI Operator. If you already have installed one of the Dell CSI driver in your cluster and  want to use the operator based deployment, uninstall the driver and then redeploy the driver following the installation procedure described above
```

Comment:
I think we should use full words in docs, please change can't to cannot or say "is unable to manage any existing driver ..."

Reply:
Done